benefits/core/templates/core/widgets/flow-radio-select.html

```diff
@@ -13,7 +13,7 @@
           {% endif %}
 
           {% for option in options %}
-            <div class="radio-input-group d-flex">{% include option.template_name with widget=option %}</div>
+            <div class="radio-input-group gap-4 gap-md-3 d-flex pt-4">{% include option.template_name with widget=option %}</div>
```

Comment:
I don't see `radio-input-group` in any CSS either. The way I'm looking for it is in the devtools Inspector:

![Screenshot from 2024-12-05 17-54-37](https://github.com/user-attachments/assets/f68ce112-2748-4f73-8171-6c3fe3d3d872)

Reply:
I suggest we remove `radio-input-group` too